src/dotnet-install.ps1

```diff
@@ -17,7 +17,7 @@
     - 2-part version in a format A.B - represents a specific release
           examples: 2.0, 1.0
     - Branch name
-          examples: release/2.0.0, Master
+          examples: release/2.0.0, Main
```

Comment:
@mmitche Are we going to start publishing bits into main folder in blob storage now? 

This url is not pointing to anything yet: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/main/latest.version

Reply:
@DavidKarlas I think we should revert this line since it doesn't work yet and there is no more time left to delay m2m transition.